docs/README.md

```diff
@@ -0,0 +1,16 @@
+# PyScript documentation
+
+Welcome to the PyScript documentation directory, where PyScript
+is and related topics are discussed.
+
+## Getting started
+
+Before you start contributing to the documentation, it's worthwhile to
+take a look at the general contributing guidelines for the PyScript project.
+
+### Setup
+
+The `docs` directory in the pyscript repository contains
+[Sphinx](https://www.sphinx-doc.org/) documentation project, which is a system
+to take a plaintext files containing the documentation written in Markdown
+and static files such as templates and themes to build the static end result.
```

Comment:
Minor adjustments. The phrase "such as templates and themes" could alternatively be between commas or parentheses:
The `docs` directory in the pyscript repository contains the
[Sphinx](https://www.sphinx-doc.org/) documentation project, a system for publishing static files—such as templates and themes—and plaintext files written in markdown.

Reply:
Ooh, yes, the em dashes read better.

Reply:
Did a slightly different rewrite. Let me know what you think once I push my changes.